neo/Consensus/ChangeView.cs

```diff
@@ -4,31 +4,49 @@ namespace Neo.Consensus
 {
     public class ChangeView : ConsensusMessage
     {
+        public enum ChangeViewReason : byte
+        {
+            Timeout = 0x0,
+            WrongExpectedView = 0x1,
+            TxNotFound = 0x2,
+            TxRejectedByPolicy = 0x3,
+            TxInvalid = 0x4,
```

Comment:
Veri gud veri naiceee, Shargon!

I like this. It is great that you are pushing this!